templates/iot/examples/infinispan-service.yml

```diff
@@ -0,0 +1,80 @@
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    description: Hot Rod's port.
+  labels:
+    application: infinispan-server
+  name: infinispan-server-hotrod
```

Comment:
This doesn't seem to math with the "example" IoTConfig, which defines `example-infinispan` instead of `infinispan-server-hotrod`.

Reply:
yes you are right, the value in the _example_ file is a remnant from the service exposed by the infinispan operator which I used during development.
thanks for catching this